modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/FilePageStore.java

```diff
@@ -565,14 +566,16 @@ private void reinit(FileIO fileIO) throws IOException {
                     if (calculateCrc && !skipCrc) {
                         assert PageIO.getCrc(pageBuf) == 0 : U.hexLong(pageId);
 
-                        PageIO.setCrc(pageBuf, calcCrc32(pageBuf, pageSize));
+                        PageIO.setCrc(pageBuf, U.calcCrc(pageBuf, pageSize));
```

Comment:
If method contract is the same, why this change?

Reply:
calcCrc32 i fully replace this method.